src/models/necessity.ts

```diff
@@ -0,0 +1,44 @@
+import dbConn from "../configs/db.config"
+import { SelectQuery } from "../db/interfaces/selectQuery";
+import { unwrapResult } from "../helpers/resultHelpers";
+import { DbNecessity, Necessity, necessityFromDb } from "../interfaces/necessity"
+
+// TODO: create endpoint for all necesities
+// TODO: create endpoint for all necesities with filters
```

Comment:
```
// TODO: create endpoint for all necesities
// TODO: create endpoint for all necesities with filters
```
Estos endpoints son el mismo. Los filtros deberían ser todos opcionales y aplicarse solo si están presentes.

Reply:
Lo sé, solo puse ambos TODOs para tenerlo como tareas separadas, porque tener una tarea que diga "create endpoint for all necessities" y otra que diga "add filters feature to endpoint for all necessities" cuando el endpoint para todas las necesidades no está hecho, me confunde un poco.